mem/template/data.template

```diff
@@ -7,7 +7,7 @@ _ATOM TYPE COUNT PLACEHOLDER_
 -25.1147367097 25.1147367097 ylo yhi
 -100.0 100.0 zlo zhi
 
-Masses
+Masses # actually overridden by the "sphere" atom_style parameters {(pi/6)*rho*d^3} - LAMMPS "bug"?
```

Comment:
If there's a bug, please start a discussion on the github, don't worry too much about adding comments about broken code, just give a file and line reference here and it'll be far easier to find later on.

Reply:
Oh... OK :) Makes sense, I'm still not used to github I guess xD Just remove the comment then...

Reply:
I can't make edits to your pull requests. I sent you some stuff on slack to have a look at.